indexing-service/src/main/java/org/apache/druid/indexing/common/actions/TaskAction.java

```diff
@@ -40,6 +40,7 @@
     @JsonSubTypes.Type(name = "segmentTransactionalReplace", value = SegmentTransactionalReplaceAction.class),
     // Type name doesn't correspond to the name of the class for backward compatibility.
     @JsonSubTypes.Type(name = "segmentListUsed", value = RetrieveUsedSegmentsAction.class),
+    @JsonSubTypes.Type(name = "segmentListLocked", value = RetrieveSegmentsToReplaceAction.class),
```

Comment:
`segmentListToReplace`?

Reply:
Can I use retrieveSegmentsToReplace since the other actions do not match the class names for backward compatibility?